.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,6 +20,7 @@ Describe the issue that you're seeing.
 ### Steps to reproduce
 
 Clear steps describing how to reproduce the issue. Please please please link to a demo project if possible, this makes your issue _much_ easier to diagnose (seriously).
+You can check how to make a minimal reproduction case [here](https://www.gatsbyjs.org/contributing/how-to-make-a-reproducible-test-case/)
```

Comment:
When users create they will see raw text in editor so using markdown link might not be advised for the template:
![Screenshot 2019-10-15 at 10 58 42](https://user-images.githubusercontent.com/419821/66816687-c6905480-ef3a-11e9-95bc-f26bc17af298.png)

Instead, (IMO),  it would be easier for users to have "standalone" link like we have above:
https://github.com/gatsbyjs/gatsby/blob/d76a62a252691f69106498afdca4dea28cab99b8/.github/ISSUE_TEMPLATE/bug_report.md#L9-L13

So something along these lines:
```suggestion

How to Make a Minimal Reproduction: https://www.gatsbyjs.org/contributing/how-to-make-a-reproducible-test-case/
```

I followed 
```
How to File an Issue: https://www.gatsbyjs.org/contributing/how-to-file-an-issue/
```
By copying documentation page title followed by link to it

Reply:
Thanks @pieh